### ISSUE TITLE
Add customer media header

### ISSUE THREAD

**Issue by Max0712** [gegis/smart-downloader#2]
Is it possible to add custom headers to the requests?

example: 
MediaHeader = {
        'Authorization': 'Basic QVBJASDjZXNzLPG1WE0wQW9qSVI6V1lreCRHVyZeJXU3Mw==',
        'content-type': 'application/octet-stream'
    };  

Othervise this seems really nice so far! :)

**Comment by gegis** [comment]
Hi,

I've just published 1.2.1 version, if you check readme docs, you should see, that it now accepts headers array, as well as other wget options, 
Headers example:
```
const SmartDownloader = require('smart-downloader');
const downloader = new SmartDownloader();
downloader.download({
    uri: 'https://github.com/gegis/smart-downloader/raw/master/test/fixtures/code.tar.gz',
    destinationDir: './downloads/',
    extractDir: './downloads/code',
    headers: ['Accept-Language: "en-us"', "Accept-Encoding: 'gzip, deflate'"]
}, (err, data) => {
    if (err) {
        console.log(err);
    }
    console.log(data);
});
```
or other custom wget options:
```
const SmartDownloader = require('smart-downloader');
const downloader = new SmartDownloader();
downloader.download({
    uri: 'https://github.com/gegis/smart-downloader/raw/master/test/fixtures/code.tar.gz',
    destinationDir: './downloads/',
    extractDir: './downloads/code',
    wgetOptions: ['--no-dns-cache', '--wait=1']
}, (err, data) => {
    if (err) {
        console.log(err);
    }
    console.log(data);
});
```

If you could please test it with your case and let me know if that works.

**Comment by Max0712** [comment]
Awesome, thank you! :)
